boilerplate/app/components/Screen.tsx

```diff
@@ -159,12 +160,17 @@ function ScreenWithScrolling(props: ScreenProps) {
     style,
   } = props as ScrollScreenProps
 
+  const ref = useRef<ScrollView>()
+
   const { scrollEnabled, onContentSizeChange, onLayout } = useAutoPreset(props as AutoScreenProps)
 
+  // Add native behavior of pressing the active tab to scroll to the top of the content
+  // More info at: https://reactnavigation.org/docs/use-scroll-to-top/
+  useScrollToTop(ref)
+
   return (
     <ScrollView
-      keyboardShouldPersistTaps={keyboardShouldPersistTaps}
-      scrollEnabled={scrollEnabled}
+      {...{ keyboardShouldPersistTaps, scrollEnabled, ref }}
```

Comment:
I'm fine with reverting this back, but for a bunch of props that just pass the same name/var the shorthand can look less busy when reading through the render function.

Reply:
I actually kinda like this. Will try using more.